tools/clang/test/SemaHLSL/hlsl/workgraph/dispatch_grid.hlsl

```diff
@@ -0,0 +1,126 @@
+// RUN: %dxc -Tlib_6_8 -verify %s
```

Comment:
None of the tests here verify the `uint16_t` case.

Reply:
Added uint16_t and int16_t cases.